Sources/BraintreePayPal/BTPayPalAppSwitchReturnURL.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 enum BTPayPalAppSwitchReturnURLState {
     case unknown
```

Comment:
Take it or leave it - instead of unknown, can this be more descriptive? Ex: `malformedURL`, `unknownStatus`, `missingPath`

Reply:
✅ : https://github.com/braintree/braintree_ios/pull/1237/commits/aa297db183ab668cd64e0c416348059aab83eaad